(node:57091) Error: This was rejected
    at Object.<anonymous> (/usr/local/google/home/yangguo/node/test/message/unhandled_promise_trace_warnings.js:4:26)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Function.Module.runMain (module.js:609:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:598:3
(node:57091) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitWarning (internal/process/promises.js:71:15)
    at emitPendingUnhandledRejections (internal/process/promises.js:88:11)
    at runMicrotasksCallback (internal/process/next_tick.js:124:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:611:11)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:598:3
(node:57091) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
    at getAsynchronousRejectionWarningObject (internal/process/promises.js:12:10)
    at rejectionHandled (internal/process/promises.js:44:21)
    at process.<anonymous> (internal/process/promises.js:23:7)
    at Promise.catch (<anonymous>)
    at Immediate.setImmediate [as _onImmediate] (/usr/local/google/home/yangguo/node/test/message/unhandled_promise_trace_warnings.js:5:27)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
